src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -25,8 +25,7 @@ public class ConvertToJsonCommand : PSCmdlet
         [AllowNull]
```

Comment:
@iSazonov, your last commit had 3 failures in `PowerShell-CI-macos`
Json Bug fixes.ConvertTo-Json -Depth 100 throws JsonException (CycleDetected) when the user specifies a depth greater or equal than 100.
```powershell
Expected strings to be the same, but they were different.
Expected length: 54
Actual length:   13
Strings differ at index 0.
Expected: 'ThrowInval...'
But was:  'ConvertToJson'
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Json.Tests.ps1: line 1411
1411:                 $exc.Exception.TargetSite.Name | Should -BeExactly $testCase.TargetSiteName
```
ConvertTo-Json.Parameter works: -Depth <depth>
```powershell
Expected strings to be the same, but they were different.
Expected length: 54
Actual length:   13
Strings differ at index 0.
Expected: 'ThrowInval...'
But was:  'ConvertToJson'
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1: line 118
118:         $exc.Exception.TargetSite.Name | Should -BeExactly "ThrowInvalidOperationException_SerializerCycleDetected"
```
ConvertTo-Json.Cycle detection works
```powershell
Expected strings to be the same, but they were different.
Expected length: 54
Actual length:   13
Strings differ at index 0.
Expected: 'ThrowInval...'
But was:  'ConvertToJson'
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1: line 176
176:         $exc.Exception.TargetSite.Name | Should -BeExactly "ThrowInvalidOperationException_SerializerCycleDetected"
```